packages/mask/src/plugins/Trader/constants/uniswap.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @dimensiondev/no-bigint */
```

Comment:
Seems like your rule has a false-positive. Only `BigInt(1)` or `1n` is the usage of bigint, `JSBI.BigInt(50)` is not